src/main/java/entities/Comment.java

```diff
@@ -8,14 +9,17 @@
 
 
 public class Comment extends Postable{
-
+    // Class Comment extends abstract class Postable and inherits all of its instance
+    // attributes (user, body, replies) and addComment method
+    // (which has already been implemented)
     public static Date creationDate;
```

Comment:
Since we are adding comments as replies to the Post, I think we will have to make sure to not accidentally add the same comment twice. Could you add a system of IDs to the comments (similar to what we did for User and Post) and a .equals method that checks for ID equality? In that case, we could probably make the ID a static variable for Portable so that Comment and Post both have access to it and can use the same .equals method (unless we want to keep track of the number of Posts and Comments separately.